### PR TITLE
[17.0][MIG] account_banking_sepa_direct_debit

### PR DESCRIPTION
**Pull request by JasminSForgeFlow**
Standard Migration

@FrogeFlow

Depends on:
- https://github.com/OCA/bank-payment/pull/1231

### CODE REVIEW

File: account_banking_sepa_direct_debit/data/mandate_expire_cron.xml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8" ?>
+<?xml version="1.0" ?>
```
Comment (MiquelRForgeFlow) [inline account_banking_sepa_direct_debit/data/mandate_expire_cron.xml]: Please @JasminSForgeFlow revert this kind of change in the .xml files. The warning is for .html files.

Comment (JasminSForgeFlow) [reply account_banking_sepa_direct_debit/data/mandate_expire_cron.xml]: Updated, Thanks

**Comment by HaraldPanten** [comment]
@JasminSForgeFlow This PR is ready to rebased. On the other hand, you're modifying two different modules.

1- The migration of account_banking_sepa_direct_debit.
2- FIX for account_banking mandate.

Could you separate that in two different PR? One for the migration and the other one for the FIX?

THX

**Comment by HaraldPanten** [comment]
Hi @MiquelRForgeFlow @JasminSForgeFlow Did you have availability to check my comments?

THX!

**Comment by pedrobaeza** [comment]
Please rebase and fix CI, and include #1244

**Comment by JasminSForgeFlow** [comment]
> @JasminSForgeFlow This PR is ready to rebased. On the other hand, you're modifying two different modules.
> 
> 1- The migration of account_banking_sepa_direct_debit. 2- FIX for account_banking mandate.
> 
> Could you separate that in two different PR? One for the migration and the other one for the FIX?
> 
> THX

Done, Thanks

**Comment by JasminSForgeFlow** [comment]
> Please rebase and fix CI, and include #1244

Done, Thanks

**Review by HaraldPanten — APPROVED** [review]

**Comment by HaraldPanten** [comment]
LGTM @pedrobaeza 

Last tests should be directly in a production environment.

WDYT?

File: account_banking_sepa_direct_debit/models/mail_compose_message.py [hunk 2]
```diff
@@ -10,7 +11,7 @@ class MailComposeMessage(models.TransientModel):
     def _action_send_mail(self, auto_commit=False):
         for wizard in self:
             if self.env.context.get("is_sent"):
-                self.env[wizard.model].sudo().browse(wizard.res_id).is_sent = True
-        return super(MailComposeMessage, self)._action_send_mail(
-            auto_commit=auto_commit
-        )
+                self.env[wizard.model].sudo().browse(
+                    json.loads(wizard.res_ids)
```
Comment (pedrobaeza) [inline account_banking_sepa_direct_debit/models/mail_compose_message.py:15]: Why do you need to convert it to JSON?

Comment (JasminSForgeFlow) [reply account_banking_sepa_direct_debit/models/mail_compose_message.py:15]: It is returning list in string format that's why

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**